src/main/resources/data/forge/tags/items/ingots/osmium.json

```diff
@@ -0,0 +1,6 @@
+{
+  "replace": false,
+  "values": [
+    "#forge:ingots/osmium"
```

Comment:
`#forge:ingots/osmium` is a reference to a forge tag (same), you need to use `janoeo:osmium_ingot`